frontend-web/webclient/app/Project/Subprojects.tsx

```diff
@@ -113,8 +116,14 @@ const SelectableWallet: React.FunctionComponent<{
                         )}
                         {props.quotaInTotal === undefined ? null : (
                             <tr>
-                                <th>Quota</th>
-                                <td>{props.quotaInTotal === -1 ? "No quota" : sizeToString(props.quotaInTotal)}</td>
+                                <th>Used/Quota</th>
+                                <td>{sizeToString(props.usage ?? 0)}
+                                    {" "}/{" "}
+                                    {sizeToString(props.remainingQuota!!)}
+                                    {" "}({(100 * (props.remainingQuota !== 0 ?
+                                            ((props.usage ?? 0) / props.remainingQuota!!) : 1
```

Comment:
One bang is enough

Reply:
I think maybe `props.remainingQuota ? /* */` would work, as it considers `null`, `undefined` and `0` as false.